psst-gui/src/data/migration.rs

```diff
@@ -0,0 +1,109 @@
+use std::{fs, path::Path};
+
+use directories::ProjectDirs;
+use platform_dirs::AppDirs;
+
+use crate::data::config::Config;
+
+const APP_NAME: &str = "Psst";
+
+pub fn perform_migration() {
+    let old_app_dirs = AppDirs::new(Some(APP_NAME), false);
+    let new_project_dirs = ProjectDirs::from("", "", APP_NAME);
+
+    if let (Some(old_dirs), Some(new_dirs)) = (old_app_dirs, new_project_dirs) {
+        migrate_path(
+            &old_dirs.config_dir,
+            new_dirs.config_dir(),
+            &["config.json"],
+        );
+        migrate_path(
+            &old_dirs.cache_dir,
+            new_dirs.cache_dir(),
+            &[
+                "tracks",
+                "track",
+                "episodes",
+                "episode",
+                "audio",
+                "keys",
+                "key",
+                "lyrics",
+                "images",
+                "artist-info",
+                "related-artists",
+                "album",
+                "show",
+                "user-info",
+            ],
```

Comment:
```suggestion
                "user-info",
                "country_code",
            ],
```

Country code is cached by `psst-core`... Honestly it feels weird that it's a tiny 2B file cached like this, but that's a story for another day.